Source/WebCore/plugins/PluginData.h

```diff
@@ -113,6 +113,10 @@ class PluginData : public RefCounted<PluginData> {
 
     String pluginFileForWebVisibleMimeType(const String& mimeType) const;
 
+    std::optional<PluginInfo> builtInPDFPlugin() const { return m_builtInPDFPluginInfo; }
```

Comment:
Should return a `const std::optional<PluginInfo>&` to avoid copying whenever possible.

Reply:
Fixed.